### PR TITLE
[DO-1579] Quality investigations page

### PR DESCRIPTION
**Pull request by viannbreslerAccso**
Implemented quality investigations figma designs
Added table settings component to table and parts-table components
Fixed issue with incorrection item counts in quality alerts and quality investigates (send and received tabs)

### CODE REVIEW

File: frontend/src/assets/locales/de/common.json [hunk 1]
```diff
@@ -71,6 +71,13 @@
       "shellDescriptorsFetchDelta": "Abruf-Delta",
       "endDate": "Enddatum"
     },
+    "tableSettings": {
+      "title": "Einstellungen der Spalte",
+      "selectAll": "Alle",
+      "orderTooltip": "Die Anordnung der Tabellenspalten kann über die Auswahl einer Spalte und einem Klick auf die Pfeilsymbole angepasst werden. ↑ (hoch) oder ↓ (runter)",
```
Comment (JoelDietz) [inline frontend/src/assets/locales/de/common.json:77]: Does not exist any longer so please remove or add it back in to the drag and drop indicators. (But then modify the tool tip)

File: frontend/src/assets/locales/en/common.json [hunk 2]
```diff
@@ -71,6 +71,13 @@
       "shellDescriptorsFetchDelta": "Fetch Delta",
       "endDate": "End date"
     },
+    "tableSettings": {
+      "title": "Column settings",
+      "selectAll": "All",
+      "orderTooltip": "Table column order can be changed by selecting a column and move it up or down by clicking on the icon ↑ (up) or ↓ (down)",
```
Comment (JoelDietz) [inline frontend/src/assets/locales/en/common.json:77]: Does not exist any longer so please remove or add it back in to the drag and drop indicators. (But then modify the tool tip)

**Review by JoelDietz — CHANGES_REQUESTED** [review]
The following functional things did I find:
1. AsPlanned table on the my parts page does not have the settings column.

2. The settings gear does not work on the quality alerts queued and sent table. (ERROR TypeError: tableSettingsService.getStoredTableSettings()[this.tableType] is undefined)

3. When changing settings on the quality investigations and quality alerts pages received tab it also changes the view on the queued and sent table view. The settings can be overwritten with there own settings but these get overwritten again if you change the settings on the received table.

4. The scroll bar of the page keeps its position on the right of the field and not on the viewport if you zoom out. So it wanders inward the farther you zoom out. Is this the wished behavior?

File: frontend/src/assets/locales/en/common.json (hunk 2 above)
Comment (viannbreslerAccso) [reply frontend/src/assets/locales/en/common.json]: Done

**Review by JoelDietz — APPROVED** [review]

**Pull request merged**